__mocks__/@ui5/webcomponents-react-charts.js

```
@@ -1,4 +1,4 @@
-const React = require("react");
+const React = require('react');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)